src/components/publiq-ui/Button.stories.mdx

```diff
@@ -4,10 +4,11 @@ import { Button, ButtonVariants } from './Button.js';
 <Meta title="Components/Button" />
 
 export const commonArgs = {
-  visible: true,
-  dismissible: true,
+  disabled: false,
 };
 
+export const commonArgTypes = { onClick: { action: 'clicked' } };
```

Comment:
Where is the name 'commonArgs' coming from? Aren't these the actual props of the component?